index.js

```diff
@@ -31,4 +31,158 @@ navLinks.forEach((link) => {
+      <p class="section-text">${item2.para1}</p>
+      <ul class="section-skills">
+        <li class="skills">${item2.tech[0]}</li>
+        <li class="skills">${item2.tech[1]}</li>
+        <li class="skills">${item2.tech[2]}</li>
+      </ul>
+      <a href="javascript:void(0)" class="btn-text" name="${item2.id}">See Project</a>
+    </div>
+  </div>
+  `;
+});
+
+sectionLabel.innerHTML += '<div id="popUp" class="popUp hide"></div>';
+
+const seeProject = document.querySelectorAll('.btn-text');
+const popUp = document.getElementById('popUp');
+
+seeProject.forEach((item) => {
+  item.addEventListener('click', () => {
+    const name = parseInt(item.getAttribute('name'), 10);
+
+    const card = workCards.filter((it) => {
+      if (it.id === name) {
+        return it;
+      }
+      return false;
+    });
+
+    popUp.innerHTML = `
+      <div class="popUp-content">
+        <div class="sectionLabel">
+          
+          <div class="label-inside">
+            <div class="popTitle">
+              <h2 class="section-title">${card[0].title}</h2>
+              <img class="crossblack" src="Images/Enabled.png" alt="crossblack" onclick="popUp.classList.toggle('hide')">
+            </div>
+            <div class="section-options">
+              <h3 class="optionBold">${card[0].canopy[0]}</h3>
+              <span class="dot"></span>
+              <h3 class="options">${card[0].canopy[1]}</h3>
+              <span class="dot"></span>
+              <h3 class="options">${card[0].canopy[2]}</h3>
+            </div>
+            <img class="portfolios" src=${card[0].imageMob} alt="card1">
+            <img class="desk-img popup-img" src=${card[0].imageDesk} alt="card1">
+            <div class="deskColum">
+              <p class="section-text cardParaMob">${card[0].para}</p>
+              <p class="section-text cardParaDesk">${card[0].para2}</p>
+              <div class="sectionFlex">
+                <ul class="section-skills">
+                  <li class="skills">${card[0].tech[0]}</li>
+                  <li class="skills">${card[0].tech[1]}</li>
+                  <li class="skills">${card[0].tech[2]}</li>
+                </ul>
+                <ul class="section-skills">
+                  <li class="skills cardParaDesk">${card[0].tech[3]}</li>
+                  <li class="skills cardParaDesk">${card[0].tech[4]}</li>
+                  <li class="skills cardParaDesk">${card[0].tech[5]}</li>
+                </ul>
+                <div class="borderPop"></div>
+                <div class="buttonPop">
+                  <a href="${card[0].live}" class="btn-text" name="1">See live <img class="imgButton" src="Images/live.png" alt="live" width="30"></a>
+                  <a href="${card[0].source}" class="btn-text" name="1">See Source <img class="imgButton" src="Images/IconGitHub.png" alt="IconGitHub" width="30"></a>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Good job trying to fix some of changes I mentioned in the previous review. So, I noticed that your design should improved again in to match the one from your chosen Figma design(e.g.: background color( add blur), center the popup card, fix the image width,...).

| Figma Design | Your Design |
| ------------------ | ------------------- |
| ![Screenshot from 2023-02-08 16-20-20](https://user-images.githubusercontent.com/83042742/217555899-61256e1c-e8c5-4192-a631-9899519f610b.png) | ![Screenshot from 2023-02-08 23-16-26](https://user-images.githubusercontent.com/83042742/217653766-c18b6074-e60b-4b0b-ad65-bf905d070eb6.png) |